### PR TITLE
fix:modifyTeamsCmd

### PR DESCRIPTION
**Pull request by arshad-k7**
#### Summary

Added multi error handing in modifyTeamsCmdF function
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/21471

### CODE REVIEW

File: commands/team.go [hunk 1]
```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if public {
 		privacy = model.TeamOpen
 	}
-
+	var Error error
```
Comment (noxer) [inline commands/team.go:339]:
```suggestion
	var errors *multierror.Error
```
We generally avoid upper case variable names in Go. In this case we want to collect the errors in a `*multierror.Error`.

File: commands/team.go [hunk 2]
```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
...
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
+			Error = multierror.Append(Error, fmt.Errorf("Unable to find team '" + args[i] + "'"))
```
Comment (noxer) [inline commands/team.go:343]:
```suggestion
			Error = multierror.Append(Error, fmt.Errorf("Unable to find team %q", args[i]))
```
You can use a placeholder to put the argument into the error. Most function in `fmt` that end in `f` can do this (`Printf`, `Errorf`, `Sprintf`, etc).

File: commands/team.go [hunk 3]
```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
...
 			printer.PrintError("Unable to find team '" + args[i] + "'")
 			continue
 		}
 		if updatedTeam, _, err := c.UpdateTeamPrivacy(team.Id, privacy); err != nil {
+			Error = multierror.Append(Error, fmt.Errorf("Unable to modify team '" + team.Name + "' error: " + err.Error()))
```
Comment (noxer) [inline commands/team.go:348]: Same as above, you can wrap errors with `%w`, which is specific for `fmt.Errorf`.

File: commands/team.go [hunk 4]
```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
...
 			printer.PrintError("Unable to modify team '" + team.Name + "' error: " + err.Error())
 		} else {
 			printer.PrintT("Modified team '{{.Name}}'", updatedTeam)
 		}
 	}
 
-	return nil
+	return Error
```
Comment (noxer) [inline commands/team.go:355]: After the change to `*multierror.Error`, you need to call `errors.ErrorOrNil()` to return an `error` here.

**Review by noxer — CHANGES_REQUESTED** [review]
Thank you very much for your contribution @arshad-k7. I'd like to request a few changes. Could you also please update the tests, so they check for the error. You can find the tests in `commands/team_test.go`.

**Comment by isacikgoz** [comment]
@arshad-k7 Let us know if you need any further guidance to address what @noxer pointed out. It's okay if you can't find some time to work on this. Just keep us posted.

**Comment by isacikgoz** [comment]
Closing this PR due to inactivity. Thanks for your efforts so far @arshad-k7!

**Pull request closed without merging**